src/app/tickets/ticket-list/ticket-list.component.ts

```diff
@@ -1,7 +1,15 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, ViewChild } from '@angular/core';
 import { Router, ActivatedRoute } from '@angular/router';
```

Comment:
Looks like these router imports can be removed.

Reply:
good catch